File: backend/src/main/java/com/manosgrigorakis/logisticsplatform/payments/dto/ReconciliationProcessResponse.java
package com.manosgrigorakis.logisticsplatform.payments.dto;


public record ReconciliationProcessResponse (
        Integer totalInvoices,
        Integer matchedInvoices,
        Integer unmatchedInvoices,
        Integer matchedTransactions,
        Integer unmatchedTransaction,
        Integer paidInvoices,
        Integer partiallyPaidInvoice,
        Integer outstandingInvoices,
        Integer disputedInvoices,
        ReconciliationReportCreateResponseDTO reportSummary
) {}

File: frontend/src/app/payments/models/reconciliation-process-response.ts
import { ReconiliationReportSummary } from './reconsiliation-report-summary';

export interface ReconciliationProcessResponse {
  totalInvoices: number;
  matchedInvoices: number;
  unmatchedInvoices: number;
  matchedTransactions: number;
  unmatchedTransaction: number;
  paidInvoices: number;
  partiallyPaidInvoice: number;
  outstandingInvoices: number;
  disputedInvoices: number;
  reportSummary: ReconiliationReportSummary;
}
